src/components/ont/runs/ONTRunInstrumentFlowcells.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div>
+    <traction-section number="2" title="Run Instrument Flowcells">
+      <div v-if="getInstrumentLayout.rows">
+        <div v-for="rowIndex in numOfRows" :key="rowIndex" class="flex flex-row px-2 py-2">
+          <ONTFlowcell
+            v-for="colIndex in numOfColumns"
+            :key="colIndex"
+            :position="calculatePosition(rowIndex, colIndex)"
+            :coordinate="calculateCoordinate(rowIndex, colIndex)"
+            class="flex flex-col px-2 py-2"
+          ></ONTFlowcell>
+        </div>
+      </div>
+      <div v-else>Please select an Instrument</div>
+    </traction-section>
+  </div>
+</template>
+
+<script>
+import { createNamespacedHelpers } from 'vuex'
+const { mapGetters } = createNamespacedHelpers('traction/ont/runs')
+
+import ONTFlowcell from '@/components/ont/runs/ONTFlowcell'
+/**
+ * # ONTRunInstrumentFlowcells
+ *
+ * Displays a visual representation of the selected instrument and it's flowcells.
+ * Pool panels can be dragged into the flowcells to link them.
+ */
+export default {
+  name: 'ONTRunInstrumentFlowcells',
+  components: {
+    ONTFlowcell,
+  },
+  props: {},
+  computed: {
+    ...mapGetters(['currentRun', 'instrumentFlowcellLayout', 'instruments']),
+    getInstrumentLayout() {
+      let instrumentConfig = this.instruments.find(
+        (instrument) => instrument.name == this.currentRun.instrument_name,
```

Comment:
If you set the whole instrument in the currentRun i think you could do something like currentRun.instrument.flowcellLayout instead.

Reply:
@harrietc52